scripts/pso/pso_param_generation.py

```diff
@@ -38,8 +39,14 @@
 PARAM_MAPREDUCE_MAP_JAVA_OPTS = 'mapreduce.map.java.opts'
 PARAM_MAPREDUCE_REDUCE_JAVA_OPTS = 'mapreduce.reduce.java.opts'
 
+PARAM_SPARK_EXECUTOR_MEMORY = "spark.executor.memory"
```

Comment:
You can include these with the spark changes.

Reply:
Done.